go/pkg/bertyaccount/service.go

```diff
@@ -96,7 +100,10 @@ func NewService(opts *Options) (Service, error) {
 	return s, nil
 }
 
-func (s *service) Close() error {
+func (s *service) Close() (err error) {
+	_, _, endSection := tyber.Section(context.TODO(), s.logger, "Closing AccountService")
```

Comment:
shouldn't this use something like `s.ctx` instead of `context.TODO()`?

Reply:
Well I could in this case but it's tricky
The service might have a traceID in the context and so it won't create a new trace
I wanted to be sure to create a new trace, I should add an helper for that instead of using context.TODO